bin/docker-entrypoint.sh

```diff
@@ -1,164 +0,0 @@
-#!/bin/bash
```

Comment:
This file appears unused now. I deleted it in https://github.com/10up/triton-wordpress/pull/3